README.md

```diff
@@ -1,31 +1,45 @@
 # The Node.js Foundation TSC
 
-The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
+The Node.js Foundation Technical Steering Committee is the technical governing
+body of the Node.js Foundation. It admits and oversees all top-level Projects
+in the Node.js Foundation. It also elects a representative to the Node.js
+Foundation Board of Directors.
 
-For more details read the [TSC Charter](https://github.com/nodejs/TSC/blob/master/TSC-Charter.md) adopted by the Node.js Foundation Board of Directors on June 17th 2015.
+For more details read the [TSC Charter][] adopted by the Node.js Foundation
+Board of Directors on June 17th 2015.
 
-If your project is interested in joining the Node.js Foundation please read the [Project Lifecycle.md](./Project-Lifecycle.md) documentation.
+If your project is interested in joining the Node.js Foundation please read
+the [Project Lifecycle.md](./Project-Lifecycle.md) documentation.
 
 ## TSC Scope
 
-**The TSC is responsible for Node.js Core project, a number of projects depended upon by Node.js Core, and a number of projects adjacent to Node.js Core.**
+**The TSC is responsible for Node.js Core project, a number of projects
+depended upon by Node.js Core, and a number of projects adjacent to Node.js
+Core.**
 
-The TSC exercises autonomy in managing its responsibilities and seeks agreement from the Node.js Foundation Board on any change to the scope of those responsibilities as defined below.
+The TSC exercises autonomy in managing its responsibilities and seeks agreement
+from the Node.js Foundation Board on any change to the scope of those
+responsibilities as defined below.
 
 ### Node.js Core
 
-Node.js Core is defined as the code project existing in the repository located at <https://github.com/nodejs/node> which the `node` executable and associated packages are built from, _minus_ the vendored projects included in the codebase, specifically those located in the [deps](https://github.com/nodejs/node/tree/master/deps) and [tools](https://github.com/nodejs/node/tree/master/tools) directories that are copied in from externally maintained repositories.
+Node.js Core is defined as the code project existing in the repository located
+at <https://github.com/nodejs/node> which the `node` executable and associated
+packages are built from, _minus_ the vendored projects included in the codebase,
```

Comment:
Nit: `_minus_` -> `excluding` 

(Let me know if I should just push these sorts of nits as a separate commit than you can force-push out if you don't like. I know I'm suggesting changes to text the already exists in the file.)